kolibri/plugins/coach/assets/src/views/common/commonCoachStrings.js

```diff
@@ -56,11 +56,11 @@ const coachStrings = createTranslator('CommonCoachStrings', {
   },
 
   // labels, phrases, titles, headers...
-  // activeLabel: 'Active',
-  // activeQuizzesLabel: {
-  //   message: 'Active quizzes',
-  //   context: 'An active quiz is one that is in progress.',
-  // },
+  activeLabel: 'Active',
```

Comment:
I'm confused, why are these strings re-activated? There is no concept of **Active/Inactive** any more, neither for lessons nor for quizzes... 😕 

Quizzes can be **Not started**, **Started** and **Ended**.
Lessons can be **Visible** and **Not visible** (to learners).

We van delete all the instances of _Active/Inactive_ strings when they are supposed to refer to lessons or quizzes.